gradle/maven.gradle

```diff
@@ -50,6 +51,19 @@ subprojects {
           artifact javadocJar
 
           version project.version
+
+          pom {
+            name = 'Brooklin'
+            description = 'An extensible distributed system for reliable nearline data streaming at scale'
+            url = 'https://www.github.com/linkedin/brooklin'
+
+            licenses {
+              license {
+                name = 'Copyright 2019 Linkedin, All Rights Reserved'
```

Comment:
Shouldn't the copyright year be updated to 2021? We use the current date when adding files in the repo. There's even a validation to enforce that for source files on build.

Reply:
yea, i'll update that.